src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsInformation.tsx

```diff
@@ -50,17 +55,23 @@ export const SettingsEditSettingsInformation: React.FC<SettingsEditSettingsInfor
             is: email => email !== me.email,
             otherwise: field => field.notRequired(),
           }),
-          phone: Yup.string().test(
-            "Phone must be valid",
-            "Phone number must be valid",
-            value => {
-              // https://github.com/artsy/gravity/blob/a8227694da735dc03c8ba50928325d84e4b2846b/app/models/util/phone_validation.rb#L2
-              if (!value) return true
-              const digits = (value.match(/\d/g) ?? []).length
-              const validChars = /^[+\/\-()\.\s0-9A-Za-z]+$/g.test(value)
-              return digits >= 3 && validChars
-            }
-          ),
+          phone: Yup.string()
+            .test({
+              name: "phone-number-is-valid",
+              message: "Please enter a valid phone number",
+              test: (national, context) => {
+                if (!national || !national.length) {
+                  return true
+                }
```

Comment:
@erikdstock fyi — the `PhoneNumberInput` component was fairly simple to drop in as this form was already using Formik and Yup but one thing I struggled with was making the phone attribute optional. Ended with this check to return `true` early if the value is empty.

Reply:
It's a todo on my work so that is good to hear :)

Reply:
One more update here - we're gonna take the phone input update out of scope in that linked PR since it will be much easier after merge.